tests/Adapters/Microsoft.Bot.Builder.Adapters.Twilio.Tests/files/Payload.txt

```diff
@@ -0,0 +1 @@
+ToCountry=US&ToState=CA&SmsMessageSid=SMc56324aa4a31735561b7205472192d68&NumMedia=0&ToCity=&FromZip=&SmsSid=SMc56324aa4a31735561b7205472192d68&FromState=Amba&SmsStatus=received&FromCity=&Body=Hello&FromCountry=AR&To=%2B16693421974&ToZip=&NumSegments=1&MessageSid=SMc56324aa4a31735561b7205472192d68&AccountSid=ACe4885e100ac907f66a8f694350e8f6f3&From=%2B541138050810&ApiVersion=2010-04-01
```

Comment:
I suggest saving the content of the payload file as constant readonly variable instead of storing it in a file.